templates/deployments.yaml

```
@@ -831,6 +879,8 @@ spec:
       annotations:
         checksum/config: {{ include (print $.Template.BasePath "/configmaps_st2-conf.yaml") $ | sha256sum }}
         checksum/packs: {{ include (print $.Template.BasePath "/configmaps_packs.yaml") $ | sha256sum }}
+        {{- if $.Values.secrets.st2.datastore_crypto_key }}
+        checksum/datastore-key: {{ include (print $.Template.BasePath "/secrets_datastore_crypto_key.yaml") & | sha256sum }}
```

`Helm-lint` is unhappy about this one: https://app.circleci.com/pipelines/github/StackStorm/stackstorm-ha/570/workflows/86ddf92e-4929-4456-8cac-0c76d0085ee0/jobs/2231/steps

```suggestion
        checksum/datastore-key: {{ include (print $.Template.BasePath "/secrets_datastore_crypto_key.yaml") $ | sha256sum }}
```
Looks like it's a typo here.